libs/SalesforceAnalytics/SalesforceAnalytics.xcodeproj/project.pbxproj

```diff
@@ -813,8 +815,8 @@
 				GCC_PREFIX_HEADER = "$(PROJECT_NAME)/Supporting Files/$(PROJECT_NAME)-Prefix.pch";
 				INFOPLIST_FILE = "SalesforceAnalytics/Supporting Files/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
-				"IPHONEOS_DEPLOYMENT_TARGET[sdk=macosx*]" = 14.5;
```

Comment:
The `Mac OS` ones were messed up for some reason.

Reply:
macosx was at 14.5 because that was the latest when we introduced it and we didn't test any earlier versions

Reply:
NB: It's not possible to downgrade to anything below 14.8.1 right now (it's the only 14.x still signed).

Reply:
@wmathurin Not sure I understand. Even if this says `14.0`, it should still work, as long as you have the latest `Xcode`, right? Isn't signing related to the `Xcode` version?

Reply:
Sure it will work. Although what is the point of supporting a version Apple no longer supports?

The only way for someone to still be on 14.x (with x<8.1) is to have avoided updates. It is not possible to downgrade to such versions because they are no longer signed by Apple.

Apple aggressively stops signing older version to block people for downgrading.
https://9to5mac.com/2021/10/04/apple-stops-signing-ios-14-8-for-some-devices-blocking-downgrades-from-ios-15/

Reply:
Ah, I see what you mean. Should I move to `14.8.1` across the board then?

Reply:
I'm worried about blowback from internal apps (and maybe even some external ones), if we do that.

Reply:
Also, I was considering devices in APAC and EMEA who may have declined OS updates.

Reply:
If it's going to cause trouble, then we can just go with 14.0.